docs/docs/accessibility.md

```diff
@@ -0,0 +1,320 @@
+
+### Colour contrast
+
+Ensure that all readable text on your website has sufficient colour contrast to remain maximally readable by users with low vision:
+
+- [WCAG - Understanding the Colour Contrast Requirement](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
+- [Everything About Color Contrast And Why You Should Rethink It](https://www.smashingmagazine.com/2014/10/color-contrast-tips-and-tools-for-accessibility/)
+- [A11yProject - What is Colour Contrast](http://a11yproject.com/posts/what-is-color-contrast/)
+
+It can be tedious to manually calculate the proper colour combinations for all cases in your website so instead you can [calculate an entire accessible colour palette with Colorable](http://jxnblk.com/colorable/).
+
+## Development and testing tools
+
+There are a number of tools we can use to assist in the creation of accessible web applications.
+
+### The keyboard
+
+By far the easiest and also one of the most important checks is to test if your entire website can be reached and used with the keyboard alone. Do this by:
+
+1. Plugging out your mouse.
+1. Using `Tab` and `Shift+Tab` to browse.
+1. Using `Enter` to activate elements.
+1. Where required, using your keyboard arrow keys to interact with some elements, such as menus and dropdowns. 
+
+### Development assistance
+
+We can check some accessibility features directly in our JSX code. Often intellisense checks are already provided in JSX aware IDE's for the ARIA roles, states and properties. We also 
+have access to the following tools:
+
+#### eslint-plugin-jsx-a11y
+
+The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y) plugin for ESLint provides AST linting feedback regarding accessibility issues in your JSX. Many
+IDE's allow you to integrate these findings directly into code analysis and source code windows.
+
+[Create React App](https://github.com/facebookincubator/create-react-app) has this plugin with a subset of rules activated. If you want to enable even more accessibility rules, 
+you can create an `.eslintrc` file in the root of your project with this content:
+                                                                                                      
+  ```json
+  {
+    "extends": ["react-app", "plugin:jsx-a11y/recommended"],
+    "plugins": ["jsx-a11y"]
+  }
+  ```
+
+#### react-axe
+The [react-axe](https://github.com/dylanb/react-axe) module uses [axe-core](https://github.com/dequelabs/axe-core) to report accessibility findings directly to the console while developing
+and debugging.
+
+### Testing accessibility in the browser
+
+A number of tools exist that can run accessibility audits on web pages in your browser. Please use them in combination with other accessibility checks mentioned here as they can only
+test the technical accessibility of your HTML.
+
+#### aXe by Deque
```

Comment:
Done, I have rolled all aXe related things into one section. All under the heading of `Testing accessibility in the browser`. As `axe-core` uses a browser DOM this is technically correct.